src/core/server/deprecations/deprecations_service.ts

```diff
@@ -139,9 +163,28 @@ export class DeprecationsService implements CoreService<InternalDeprecationsServ
     };
   }
 
-  public start() {}
+  public start(): InternalDeprecationsServiceStart {
+    return {
+      asScopedToClient: this.createScopedDeprecations(),
```

Comment:
Needs unit test

Reply:
Added test via https://github.com/elastic/kibana/pull/105998/commits/da04f4b6a7fb338b5c71385c74eba46212375a97. Let me know if you think there should be anything more tested here.